### PR TITLE
[Docs] Service mesh pattern management

### PR DESCRIPTION
**Pull request by leecalcote**
**Description**

This PR is provides an initial description of the upcoming pattern functionality.

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/functionality/patterns.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Take time to explore. Try deploying one service mesh’s sample application onto a different service mesh and compare differences in behavior and each service mesh’s ability to manage it. If using Meshery to do so, execute the following commands as an example to deploy the sample application, BookCatalog, onto Open Service Mesh:
+
+```
+$ mesheryctl pattern apply -f book-catalog
+Deploying application “BookCatalog”...
+Deployed. Endpoint(s) available at: http://localhost:8000/catalog
+```
+
+## Related Reading
```
Comment (alphaX86) [inline docs/pages/functionality/patterns.md:38]: What we can do here? 😅 
image

Comment (leecalcote) [reply docs/pages/functionality/patterns.md:38]: Write more docs. :)

That is a serious answer, but to your point, in the meantime until they are written, we could hide this section.

Comment (leecalcote) [inline docs/pages/functionality/patterns.md:38]:
```suggestion
<!-- ## Related Reading -->
```

Comment (leecalcote) [reply docs/pages/functionality/patterns.md]: @alphaX86, I have updated to hide this section for now.

**Pull request merged**